beacon-chain/state/state-native/state_trie.go

```diff
@@ -93,17 +93,33 @@ var denebFields = append(
 	types.HistoricalSummaries,
 )
 
+var electraFields = append(
+	denebFields,
```

Comment:
This is wrong because `denebFields` contains `LatestExecutionPayloadHeaderDeneb`